meta-lmp-base/conf/distro/include/non-clangable.inc

```diff
@@ -73,6 +73,21 @@ OBJCOPY:pn-docker-compose:toolchain-clang = "${HOST_PREFIX}objcopy"
 STRIP:pn-mcumgr:toolchain-clang = "${HOST_PREFIX}strip"
 OBJCOPY:pn-mcumgr:toolchain-clang = "${HOST_PREFIX}objcopy"
 
+# | /lmp/build-imx6ullevk/tmp-lmp/work/imx6ullevk-lmp-linux-gnueabi/linux-lmp-fslc-imx/5.15.77+gitAUTOINC+d7be55c190_9dcc132b0c-r0/temp/run.do_assemble_fitimage.520200: 841: arm-lmp-linux-gnueabi-llvm-objcopy: not found
+OBJCOPY:pn-linux-lmp-dev:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-dev-mfgtool:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-lts:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-rt:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-ea-imx:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-fslc-imx:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-fslc-imx-rt:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-rpi:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-ti-staging:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-toradex-imx:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-lmp-xlnx:toolchain-clang = "${HOST_PREFIX}objcopy"
+OBJCOPY:pn-linux-tegra:toolchain-clang = "${HOST_PREFIX}objcopy"
```

Comment:
Do you know why this started to happen?

Reply:
No, don't know where it is been introduced but can find it locally.
On main next I think see something about this on clang layer but on this stable update this makes less sense.
Anyway it fixes the build https://github.com/foundriesio/lmp-manifest/pull/284 that almost done.

Reply:
Sure, I'm ok merging this, I would just like to have a better understanding so we know when to revert later on.

Reply:
Right, I don't even know what happened just to change from one branch to another.
I will find something about this clang issues to update the commit message.

Reply:
This seems that was added in meta-clang as well:

```
+# Subprocess output:mips-yoe-linux-llvm-objcopy: error: Link field value 22 in section .rel.dyn is not a symbol table
+# also seen on riscv64 and x86-64
+OBJCOPY:pn-linux-yocto:toolchain-clang = "${HOST_PREFIX}objcopy"
```